tests/rbd/test_rbd_pwl_with_encrypted_images.py

```diff
@@ -0,0 +1,138 @@
+"""RBD Persistent write back cache with encrypted images.
+
+Test Case Covered:
+CEPH-83575409 - Pmem/SSD mode write back cache on encrypted images.
+
+Steps :
+1) create image with encryption(luks1,luks2) feature enabled
+2) Setup persistent write cache for that encrypted image
+3) Write data to the image using fio jobs check cache status
+4) Repeat the test on client, pool and image level
+
+Pre-requisites :
+- need client node with ceph-common package, conf and keyring files
+- FIO should be installed on the client.
+
+Environment and limitations:
+- The cluster should have 5 nodes + 1 SSD cache node
+- cluster/global-config-file: config/quincy/upi/octo-5-node-env.yaml
+- Should be Bare-metal.
+
+Support
+- Configure cluster with PWL Cache.
+- Only replicated pool supported, No EC pools."""
+
+from ceph.parallel import parallel
+from ceph.rbd.utils import random_string
+from tests.rbd.rbd_peristent_writeback_cache import (
+    PersistentWriteAheadLog,
+    get_entity_level,
+)
+from tests.rbd.rbd_utils import create_passphrase_file, initial_rbd_config
```

Comment:
Kindly use cli/rbd/rbd.py:initial_rbd_config for all new tests instead of old tests/rbd/rbd_utils or help me understand if we can't use cli for any reasons.

Reply:
`cli/rbd/rbd.py` don't have `initial_rbd_config` this test no need of sclaing hence i think it's ok to use existing `rbd_utils.py` as most of `pwl  cache` test uses the same.

Reply:
sorry I mean cephci/ceph/rbd/initial_config.py

is it possible to use this though most of the pwl cache uses old one ?

Reply:
@pavankumarag i tried using that `from ceph.rbd.initial_config import initial_rbd_config` since all [pwl main class](https://github.com/red-hat-storage/cephci/blob/f6f05d959133d8aa6247afd6ef32813ba3e91c07/tests/rbd/rbd_peristent_writeback_cache.py#L21) using `rbd_utils.py` so it needs full reworkflow faced lot of issues like 
```2024-02-16 06:15:24,820 (cephci.test_rbd_pwl_with_encrypted_images) [ERROR] - cephci.tests.rbd.test_rbd_pwl_with_encrypted_images.py:141 - 'Rbd' object has no attribute 'set_config'``` also lot of other existing methods causing problem which needs to be rewrite in `cli/rbd/rbd.py`
which i can take up during scale with e2e for now this test case is just `tier2` can we merge it.

Reply:
okay makes sense, we can take it up later